ui/app/templates/application.hbs

```diff
@@ -26,6 +26,8 @@
 
 <KeyboardShortcutsModal />
 
+<PortalTarget @name="log-sidebar-portal" />
```

Comment:
This is pretty high up in the DOM tree. Why not have this be a child of the `PageLayout`?

Reply:
You get `Assertion Failed: Portal target with name log-sidebar-portal already exists` at that level.